go.mod

```
@@ -5,7 +5,7 @@ go 1.12
 require (
 	github.com/davecgh/go-spew v1.1.1
 	github.com/evanphx/json-patch v4.9.0+incompatible
-	github.com/flant/shell-operator v1.0.0-beta.13.0.20201213124752-17d64f6e61d1 // branch: master
+	github.com/flant/shell-operator v1.0.0-beta.13.0.20201214165802-7644ce7cef6e // branch: master
```

```suggestion
	github.com/flant/shell-operator v1.0.0-beta.13.0.20201214204115-463c4ecd8214 // branch: master
```

More recent master to prevent merge conflict with https://github.com/flant/addon-operator/pull/160